src/yang/drned-xmnr.yang

```diff
@@ -84,6 +84,29 @@ module drned-xmnr {
          from the CLI.";
       type string;
     }
+    leaf cli-log-file {
+      tailf:info
+        "If set, all output that would be printed to CLI, goes to that
+         file; if run from a CLI session, output is printed both to
+         CLI as well as to the file.  The path should be absolute
+         (otherwise the file is opened wherever xmnr happens to be
+         running).";
+      type string;
+    }
+    tailf:action cli-log-message {
+      tailf:actionpoint xmnr-cli-log;
+      tailf:info
+        "If registered, all output that would be printed to CLI is
```

Comment:
Good point. If we decide to go with this variant, perhaps this action should be hidden or something, since it is not supposed to be used by operators.